### PR TITLE
Add y/n keyboard functionality

### PR DESCRIPTION
**Pull request by anmolm96**
Hey,
Came across this on PH and thought it was pretty cool.
Tried to implement this [feature request](https://cards.producthunt.com/cards/comments/719632).
But was running into some issues with getting the animation to work with `y` keypress.
Might need some cleanup to finish it up.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -83,9 +88,11 @@ export default () => {
                 type="confetti"
                 ref={ref => {
                   reward = ref
+                  console.log(reward)
```
Comment (semoal) [inline src/app.js]: i think console.log on prod build shouldn't appear!

Comment (anmolm96) [reply src/app.js]: Oops! Thanks for flagging. Was testing something out

**Review by SaraVieira — APPROVED** [review]

File: src/app.js [hunk 2]
```diff
@@ -48,7 +48,9 @@ export default () => {
   }
 
   const yes = () => {
-    reward.rewardMe()
+    // reward is undefined when the 'y' key is pressed.
```
Comment (SaraVieira) [inline src/app.js:51]: I had this issue too when trying to implement this :(

**Comment by SaraVieira** [comment]
Thank you so much!!

**Pull request merged**